features/support/worlds/employer_world.rb

```diff
@@ -34,3 +41,9 @@ def employer_profile
   # Need to fix below later
   employer_profile.benefit_sponsorships.first.active_broker_agency_account.update(writing_agent_id:broker.person.broker_role.id)
 end
+
+And(/^the broker has a prospect employer$/) do
```

Comment:
https://github.com/health-connector/enroll/pull/1582 We rewrote this as a legitimate step here